.github/workflows/inspektor-gadget.yml

```diff
@@ -364,13 +364,38 @@ jobs:
     # level: 1
     needs: [test-unit, build-kubectl-gadget, build-local-gadget, build-docker-image]
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        k8s-distro: [minikube-github, aro]
+    concurrency:
+      # Ensure only one test-integration job runs at a time to avoid trying to
+      # deploy IG multiple times on the same Kubernetes cluster. It is
+      # specifically needed for ARO because it is a pre-created cluster.
+      group: no-simultaneous-integration-tests
     steps:
-    - name: Setup Minikube
+    - name: Check out code
+      uses: actions/checkout@v1
+    - name: Set Kubernetes distribution
+      shell: bash
+      run: echo KUBERNETES_DISTRIBUTION="${{ matrix.k8s-distro }}" >> $GITHUB_ENV
+    - if: matrix.k8s-distro == 'minikube-github'
+      name: Setup Minikube
       uses: manusa/actions-setup-minikube@v2.4.2
       with:
         minikube version: 'v1.9.2'
         kubernetes version: 'v1.18.2'
         github token: ${{ secrets.GITHUB_TOKEN }}
+    - if: matrix.k8s-distro == 'aro'
+      name: Authenticate and set ARO cluster context
+      uses: redhat-actions/oc-login@v1
+      with:
+        # API Server URL
+        openshift_server_url: ${{ secrets.OPENSHIFT_SERVER }}
+
+        # Credentials (TODO: Use a functional Service Account)
+        openshift_username: ${{ secrets.OPENSHIFT_USER }}
+        openshift_password: ${{ secrets.OPENSHIFT_PASSWORD }}
```

Comment:
Just wondering, is it the piece that configures the kubeconfig so kubect-gadget commands are associated to that cluster?

Reply:
Yeah, it is.